src/main/webapp/app/pages/curation/collapsible/MutationCollapsible.tsx

```diff
@@ -110,6 +111,24 @@ const MutationCollapsible = ({
     );
   }, [relatedAnnotationResult]);
 
+  const associatedGenomicIndicatorUuids = useMemo(() => {
+    if (!isGermline || !genomicIndicators) {
+      return [];
+    }
+
+    const associatedIndicators: string[] = [];
+    for (const genomicIndicator of genomicIndicators) {
+      for (const variant of genomicIndicator.associationVariants || []) {
+        if (mutationUuid === variant.uuid) {
+          associatedIndicators.push(genomicIndicator.name_uuid);
+        }
+      }
+    }
+    return associatedIndicators;
+  }, [genomicIndicators, mutationUuid]);
+
+  const isAssociatedWithGenomicIndicator = associatedGenomicIndicatorUuids.length > 0;
```

Comment:
I don't think this will react when associatedGenomicIndicatorUuids is updated using the memo.